client/src/App.js

```diff
@@ -56,7 +57,7 @@ class App extends Component {
         <Navbar updateUser={this.updateUser} loggedIn={this.state.loggedIn} />
         {/* greet user if logged in: */}
         {this.state.loggedIn &&
-          <p>Join the party, {this.state.username}!</p>
+          <p>Logged In as user: {this.state.username}</p>
         }
         {/* Routes to different components */}
```

Comment:
Is this necessary comment? I'm sure the Routes tag speak for itself.

Reply:
True.

Reply:
Removed.